packages/api-server/src/app/cluster.ts

```diff
@@ -5,12 +5,15 @@ import { logger } from "../base/logger";
 import { BlockEmitter } from "../block-emitter";
 import { CKBPriceOracle } from "../price-oracle";
 import { initSentry } from "../sentry";
+import { startOpentelemetry } from "../opentelemetry";
 
 const numCPUs = cpus().length;
 const clusterCount = +(envConfig.clusterCount || 0);
 const numOfCluster = clusterCount || numCPUs;
 
-if (cluster.isMaster) {
+startOpentelemetry();
+
+if (cluster.isPrimary) {
```

Comment:
We should update it later, for `Dockerfile` is still using node 14, and this attribute added since node 16.

Reply:
Fixed